### DISCUSSION TITLE
Glitch with buttons after default viewport size change

### DISCUSSION THREAD

**Discussion by sandro78** [wKich/creevey#181] [General]
Hi guys.

Really great tool - thank you! Faced with strange glitch after change default chrome viewport from 1024x720 to 1280x1024:
<img width="1459" alt="Снимок экрана 2021-09-13 в 21 09 24" src="https://user-images.githubusercontent.com/35001759/133254654-629e42df-f5a4-4777-a34b-b669d2387ad6.png">
The only workaround i found to set captureElement=null in order to get not only modal dialog but whole viewport instead. Any clue why this could happen?

**Comment by wKich** [comment]
@sandro78 Hi, I'm sorry for the late response. That's really weird. How did you change the viewport, by defining it in creevey's config? I thought that it could happen because of how composite images (when captureElement is outside a viewport) are captured, but I don't think that modal doesn't fit in the viewport.

**Reply by sandro78** [reply]
@wKich Hi Dmitry, this is our **.creevey/config.js** content:
```
module.exports = {
  browsers: {
    chrome: {
      browserName: 'chrome',
      // Define initial viewport size
      viewport: { width: 1280, height: 1024 }
    }
  }
};
```
